### PR TITLE
Add example apps for use-case demos

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add example quote & meme services.

Quote:
Screen Shot 2022-08-17 at 15 00 58

Meme:

Screen Shot 2022-08-17 at 14 57 41

## Testing

It's described in the README.md

### CODE REVIEW

File: examples/src/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Meme & Quote
+
+Example services contains following services:
```
Comment (ezodude) [inline examples/src/README.md:3]:
```suggestion
An example project that contains the following services:
```

File: examples/src/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- **meme** - generate random memes. It takes a random quote from quote service and displays it on a random image.
+- **quote** - generate random quotes.
+
```
Comment (pkosiec) [inline examples/src/README.md:7]: It would be good to mention prerequisites (Go, Docker)

File: examples/src/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+## Local
```
Comment (pkosiec) [inline examples/src/README.md:8]: WDYT:

```suggestion
## Local usage

To run the services locally, follow the steps:
```

File: examples/src/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    ```
+    Site can be viewed at [http://localhost:8080/quote](http://localhost:8080/quote)
+
+2. Start the meme service
+    ```
+    env QUOTE_URL=http://localhost:8080 go run cmd/meme/main.go
+    ```
+    Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme)
+
+## Docker
```
Comment (pkosiec) [inline examples/src/README.md:22]:
```suggestion
## Docker usage
```

File: examples/src/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+### Build
+
```
Comment (pkosiec) [inline examples/src/README.md:25]:
```suggestion

To build the Docker images, run the following commands:

```

File: examples/src/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+2. Start the meme service
+    ```
+    env QUOTE_URL=http://localhost:8080 go run cmd/meme/main.go
+    ```
+    Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme)
+
+## Docker
+
+### Build
```
Comment (pkosiec) [inline examples/src/README.md:24]:
```suggestion
### Build images
```

File: examples/src/README.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+```bash
+env GOOS=linux GOARCH=amd64 go build -o meme ./cmd/meme
+docker build -t meme:1.0.0 -f build/Dockerfile-meme .
+
+env GOOS=linux GOARCH=amd64 go build -o quote ./cmd/quote
+docker build -t quote:1.0.0 -f build/Dockerfile-quote .
+```
+
+### Testing
```
Comment (pkosiec) [inline examples/src/README.md:34]: 🤔
```suggestion
### Run images

To run the built Docker images, run the following commands:
```

File: examples/src/README.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    ```
+    go run cmd/quote/main.go
+    ```
+    Site can be viewed at [http://localhost:8080/quote](http://localhost:8080/quote)
+
+2. Start the meme service
+    ```
+    env QUOTE_URL=http://localhost:8080 go run cmd/meme/main.go
+    ```
+    Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme)
```
Comment (pkosiec) [inline examples/src/README.md:20]:
```suggestion
    Site can be viewed at [http://localhost:8080/quote](http://localhost:8080/quote).

2. Start the meme service:
    ```
    env QUOTE_URL=http://localhost:8080 go run cmd/meme/main.go
    ```
    Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme).
```

File: examples/src/README.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+docker network create testing
+
+docker run --name quote -d  --network testing --network-alias quote -p 8080:8080 quote:1.0.0
+
+docker run --name meme --network testing -e QUOTE_URL=http://quote:8080 -p 9090:9090 meme:1.0.0
+```
+
+Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme)
+
+#### Cleanup
```
Comment (pkosiec) [inline examples/src/README.md:46]:
```suggestion
Site can be viewed at [http://localhost:9090/meme](http://localhost:9090/meme).

To clean up, run:
```

**Review by pkosiec — APPROVED** [review]
Just some minor comments 👍 

Later we can think about changing the faces to some famous figures (like e.g. Albert Einstein, Abraham Lincoln, Steve Jobs, Paolo Coelho, Ghandi?) and generate something like this:

image

or

image

Most of the quotes could be kept as they are, and the `meme` service could append a name of a figure to the final picture.
I think it would be funnier, but of course that's very subjective 😄

**Pull request merged**